### DISCUSSION TITLE
Packages requested

### DISCUSSION THREAD

**Discussion by peuportier** [open-webui/open-webui#6622] [General]
Hello everyone,

Possible to add the packaging for pyodide please. 

Networkx
statsmodels
biopython
lxml
tqdm
Pillow (PIL)
html5lib
asyncio
markdown
jinja2
dateutil
ast
plotly
xlrd (2.0.1)
yt (4.3.0)
altair (5.3.0)
astropy (6.0.1)
geopandas (0.14.3)
Cartopy (0.23.0)
pdfplumber

would be perfect 

thanks you

**Comment by mafzaal** [comment]
I have added the packages find code [here](https://github.com/mafzaal/open-webui/tree/pyodide-additional-packages). I could not make following package work, will require more research.

pdfplumber
plotly
